internal/cmd/beta/server/backup/create/create.go

```diff
@@ -0,0 +1,171 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	cliErr "github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/serverbackup/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/serverbackup"
+)
+
+const (
+	backupNameFlag            = "name"
+	backupRetentionPeriodFlag = "retention-period"
+	backupVolumeIdsFlag       = "volume-ids"
+	serverIdFlag              = "server-id"
+
+	defaultRetentionPeriod = 14
+	defaultVolumeIds       = ""
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	ServerId              string
+	BackupName            string
+	BackupRetentionPeriod int64
+	BackupVolumeIds       string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a Server Backup",
+		Long:  "Creates a Server Backup.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a Server Backup with name "mybackup"`,
+				`$ stackit beta server backup create --server-id xxx --name=mybackup`),
+			examples.NewExample(
+				`Create a Server Backup with name "mybackup" and retention period of 5 days`,
+				`$ stackit beta server backup create --server-id xxx --name=mybackup --retention-period=5`),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a Backup for server %s?", model.ServerId)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req, err := buildRequest(ctx, model, apiClient)
+			if err != nil {
+				return err
+			}
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create Server Backup: %w", err)
+			}
+
+			return outputResult(p, model, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().VarP(flags.UUIDFlag(), serverIdFlag, "s", "Server ID")
+	cmd.Flags().StringP(backupNameFlag, "b", "", "Backup name")
+	cmd.Flags().Int64P(backupRetentionPeriodFlag, "d", defaultRetentionPeriod, "Backup retention period (in days)")
+	cmd.Flags().StringP(backupVolumeIdsFlag, "i", defaultVolumeIds, "Backup volume ids, as comma separated UUID values.")
```

Comment:
```suggestion
	cmd.Flags().StringP(backupVolumeIdsFlag, "i", defaultVolumeIds, "Backup volume IDs, as comma separated UUID values.")
```

Reply:
Actually, we have a custom `UUIDSliceFlag` type (you can search for occurences in the project) which is a string slice flag that validates if all elements are UUIDs

Reply:
I think it would be the correct type for this flag, but if not it should at least be a `StringSlice` flag